main.c

```diff
@@ -747,6 +870,24 @@ static int dev_reconfig(struct tcmu_device *dev, struct tcmulib_cfg_info *cfg)
 	}
 }
 
+static void parse_tcmu_runner_args(struct tcmu_device *dev)
+{
+	struct tcmur_device *rdev = tcmu_dev_get_private(dev);
+	char *next_opt, *cfg_str;
+
+	cfg_str = strdup(tcmu_dev_get_cfgstring(dev));
+
+	next_opt = strtok(cfg_str, ";");
+	while (next_opt) {
+		if (!strncmp(next_opt, "tcmur_cmd_time_out=", 19)) {
+			rdev->cmd_time_out = atoi(next_opt + 19);
```

Comment:
@mikechristie 

BTW, just removing the "tcmur_cmd_time_out=VALUE" string from "dev->cfgstring" here makes sense ? IMHO, this timeout option shouldn't visible by the handlers.

Or in the handler like the glfs,etc which has no any extra option needed. we must handle it separately in each handler, which the code are duplicated.

Reply:
You are right. I will have that code remove it so handlers do not see it.

Reply:
This should be fixed in the repush. I also added some info in the README. The format is:

/handler type;tcmu runner arg1;tcmu runner arg2;handler args

and now the tcmu runner agrs and the ";" will be removed for the handler.

Reply:
Will test it.
Thanks.